NEWS

```diff
@@ -7,13 +7,183 @@
 ##
 #
 
-## Current
+## 1.12.0 RC1 (December 2, 2021)
 ### Features:
+#### Core
+* Added beta-level support for Go language bindings
+* Added new objects to VFS (md, component, log_level, etc)
+* Added configuration variable to specify which loadable modules are allowed
+* Added build-time configuration to disable sigaction overriding
 #### UCP
-* Added API for querying UCP library attributes
+* Added address versioning to correctly preserve wire compatibility since v1.11.0
+* Added new client/server connection establishment packet header format
+* Enabled rendezvous and tag sync protocols when error handling is enabled on the endpoint
+* Added iov zcopy support to RMA operations
+* Reduced memory usage of unexpected messages by fitting receive buffer size to packet size
+* Added support for modifying UCT and UCS configs by ucp_config_modify() API
+* Optimized unpacked rkeys memory consumption
+* Added request flag to influence latency vs. bandwidth protocol
+* Reduced memory management overhead with new protocols
+* Improved performance calculations for new protocols
+* Added AMO support with GPU memory target using new protocols
+* Added put_zcopy, get_zcopy and pipeline based rendezvous in new protocols
+* Added support for user-defined alignment in Active Messages
+* Added support for offload tag sync in new protocols
+* Updated ucp_atomic_post() to use NBX flow
+##### API
+* Added client_id to ucp_worker_create() and ucp_conn_request_query() APIs
+* Added ucp_worker_address_query() API
+* Updated ucp_ep_query() API with getting local and remote addresses
+#### UCT
+* Added API - uct_iface_is_reachable_v2()
+* Added IPv6 address support in TCP
+* Added latency estimation to uct_iface_estimate_perf()
+* Adjusted knem and cma overhead cost
+* Increased built-in TCP keep-alive interval to 2 seconds
+#### RDMA CORE (IB, ROCE, etc.)
+* Added check for CQ overrun in assert mode
+* Added bitmap usage for releasing detached DCIs
+* Added configuration for requests ack frequency with DevX
+* Added remote QP info to tx error CQE traces
 #### UCS
-* Added API to Read boot ID value or use machine_guid
+* Added memory pool set data structure
+* Improved accuracy of the topology distance estimation
+* Added thread-safe put to ptr_map
+* Added prints of leaked callbacks from the callback queue
+* Added new ptr_array API for bulk allocation
+* Added ucs_string_buffer_append_flags() for string buffer
+* Added ucs_ffs32()
+* Removed a diagnostic message when fuse thread is stopped
+* Added ucs_vsnprintf_safe() which always adds '\0'
+* Added API for a per-process aggregate-sum statistics report
```

Comment:
Please move API changes to the top of UCS section

Reply:
moved